spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -350,4 +351,23 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         }
 
     }
+    
+    @Override
+    public List<String> deleteMultipleItems(String groupId, List<String> citationKeys, Long citationVersion) throws ZoteroConnectionException {
```

Comment:
let's add this method and the one above over the class, the class should be last.

Reply:
this method should encapsulate the return values from zotero; I would return a list of enums that represent the status codes (SUCCESS, LIBRARY_LOCKED,...). Otherwise, if zotero changes something with the return codes, every caller of this method will have to be changed instead of just this method.